app/utils/uuid_utils.rb

```diff
@@ -0,0 +1,12 @@
+module UUIDUtils
+  module_function
+
+  def bin16_uuid
+    # TODO: Use https://github.com/assaf/uuid
+    s = SecureRandom.uuid.gsub("-", "")
```

Comment:
@ovan That's exactly why there is the TODO comment above. These are indeed v4 UUIDs and should be changed to use the linked library or something else.